components/style/build.rs

```diff
@@ -62,3 +71,11 @@ fn main() {
         exit(1)
     }
 }
+
+fn main() {
+    println!("cargo:rerun-if-changed=build.rs");
+    generate_properties();
```

Comment:
That is done in build_gecko.rs which recursively adds header files which it may depend on to `cargo:rerun-if-changed`.